lib/Conversion/StableHLOToTTIR/StableHLOToTTIRPatterns.cpp

```diff
@@ -1348,10 +1348,37 @@ class StableHLOToTTIRConvolutionOpConversionPattern
             : rewriter.getDenseBoolArrayAttr(
                   SmallVector<bool>(numSpatialDims, false));
 
-    ttir::utils::replaceOpWithNewDPSOp<mlir::tt::ttir::ConvolutionOp>(
-        rewriter, srcOp, outputType, adaptor.getLhs(), adaptor.getRhs(),
-        Value(), windowStridesAttr, paddingAttr, inputDilationAttr,
-        kernelDilationAttr, windowReversalAttr,
+    bool hasNegativePadding = llvm::any_of(paddingArray, [](int64_t p) { return p < 0; });
```

Comment:
:warning: **clang\-diagnostic\-error** :warning:
use of undeclared identifier `` paddingArray ``; did you mean `` paddingAttr ``?
```suggestion
    bool hasNegativePadding = llvm::any_of(paddingAttr, [](int64_t p) { return p < 0; });
```